src/main/scala/com/ing/wbaa/airlock/proxy/provider/aws/SignatureHelpersV2.scala

```diff
@@ -0,0 +1,106 @@
+package com.ing.wbaa.airlock.proxy.provider.aws
+
+import java.util
+
+import akka.http.scaladsl.model.HttpRequest
+import com.amazonaws.DefaultRequest
+import com.amazonaws.auth.BasicAWSCredentials
+import com.ing.wbaa.airlock.proxy.data.{ AWSHeaderValues, RequestId }
+import com.ing.wbaa.airlock.proxy.handler.LoggerHandlerWithId
+import com.ing.wbaa.airlock.proxy.provider.aws.SignatureHelpersCommon.extractHeaderOption
+
+import scala.collection.JavaConverters._
+
+class SignatureHelpersV2 extends SignatureHelpersCommon {
+  private val logger = new LoggerHandlerWithId
+  final val AWS_SIGN_V2 = "v2"
+
+  def getSignedHeaders(authorization: String): String = throw new Exception("V2 signature protocol doesn't support SignedHeaders")
+
+  def extractRequestParameters(httpRequest: HttpRequest, version: String): util.Map[String, util.List[String]] = {
```

Comment:
```suggestion
  def extractRequestParameters(httpRequest: HttpRequest): util.Map[String, util.List[String]] = {
```
version is not needed - we know it is v2